content/root/platform-customization.textile

```diff
@@ -0,0 +1,147 @@
+
+This ensures that under adverse conditions, such as a denial of service attack or a major outage, Ably can proactively route Enterprise customer traffic to specific data centers or regions that are unaffected, or even to new temporary clusters. This enables customers to maintain service while the issue is investigated and resolved.
+
+This service is ideal for customers who want the piece of mind that Ably actively monitors not only the global cluster health but also the endpoints used by that customer, and can act to isolate or move traffic, ensuring business continuity.
+
+h2(#dedicated-and-isolated-clusters). Dedicated, isolated clusters
+
+As part of our Enterprise package offering, we are able to offer customers dedicated clusters that are either partially or completely isolated from Ably's global cluster.  Customers who require isolation from our global cluster for security, governance or guaranteed capacity reasons are able to run the Ably platform on their own EC2 dedicated environments.  
+
+The levels of isolation Ably can offer, from least to most isolated, are:
+
+- Dedicated realtime and routing servers, but shared database cluster and common services (such as stats, logging, registries, configuration)
+- Dedicated realtime and routing servers, dedicated database cluster and shared common services
+- Dedicated realtime and routing servers, dedicated database and common services
+ 
+Ably can run a dedicated cluster on behalf of our Enterprose customers, or if required, directly on instances provided to us by our customers using their Amazon EC2 account.
+
+We provide you with custom endpoints for your cluster, for example: @yourcompany-realtime.ably.io@ and @yourcompany-realtime.ably.io@.  Additionally, we can support CNAMEs using your own custom root domain.
+
+Please note the following:
+- Ably only supports Amazon EC2 environments (within Virtual Private Networks)
+- Ably is responsible for pro-actively managing and updating dedicated clusters
+- Dedicated clusters can run in one or more regions of your choice
+ 
+If you would like to find out more about our dedicated cluster offering, please "get in touch":https://ably.com/contact.
+
+h2(#regional-routing-and-storage). Regional routing and storage
+
+If you need your data to remain within either Europe or the US, we can customize your account so that all of your data (both stored data and transient realtime data transmission activity) is kept in servers within the EU or US.
+
+Note that this does not prevent customers outside of your chosen region from connecting to Ably - it just means that, regardless of their location, they will connect to a datacenter in that region.
+
+h2(#setting-up-a-custom-environment). Setting up a custom environment
+
+To use a custom environment, perform the following steps:
+
+h3. 1. Decide whether to use an Ably subdomain, or your own domain
+
+Ably client libraries connect to two endpoints by default:
```

Comment:
is it worth including MQTT and SSE endpoints as a minor point. It came up recently and might make these a bit more accessible

Reply:
@markcoxsmith: I believe the default MQTT endpoint is `mqtts:mqtt.ably.io`, but the SSE endpoint is based on `realtime.ably.io` ... is that correct?